applications/unit_tests/detail/command_group.cpp

```diff
@@ -17,7 +17,7 @@ TEST_CASE("CommandGroup")
     auto* group = helpers::defaultCommandGroup(device, helpers::availableQueueType(adapter), 10);
 
     const auto nodeCount = adapter->queryNodeCount();
-    for (uint32_t i = 0; i < nodeCount; i++)
+    for (size_t i = 0; i < nodeCount; i++)
```

Comment:
nodeCount is a uint8_t, and I is not used to iterate, so this should prolly still be a uint8_t